src/components/EventCard.tsx

```diff
@@ -51,24 +51,44 @@ const EventCard = ({
     return <EventTag tag={t} />;
   });
 
+  let [isRegistered, setisRegistered] = useState(true);
```

Comment:
I'd recommend using a `const` here unless the variable will change, because this will enforce the variable to stay constant

Reply:
Also would name this `setIsRegistered`